fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -642,6 +644,12 @@ private OptExpression pushDownAggregation(OptExpression tree, TaskContext rootTa
         return tree;
     }
 
+    private void skewJoinOptimize(OptExpression tree, TaskContext rootTaskContext) {
+        SkewJoinOptimizeRule rule = new SkewJoinOptimizeRule();
+        Utils.calculateStatistics(tree, rootTaskContext.getOptimizerContext());
```

Comment:
what if collect statistics failed, should we do SkewJoinOptimizeRule?

Reply:
Yes, if user use hint, we still need to use SkewJoinOptimizeRule